docs/source/markdown/podman-pod-logs.1.md

```diff
@@ -0,0 +1,92 @@
+% podman-pod-logs(1)
+
+## NAME
+podman\-pod\-logs - Displays logs for pod with one or more containers
+
+## SYNOPSIS
+**podman pod logs** [*options*] *pod*
+
+## DESCRIPTION
+The podman pod logs command batch-retrieves whatever logs are present with all the containers of a pod. Pod logs can be filtered by container name or id using flag **-c** or **--containername** if needed.
+
+Note: Long running command of `podman pod log` with a `-t` needs to be reinvoked if new container is added to the pod dynamically otherwise logs of newly added containers would not be visible in log stream.
+
+## OPTIONS
+
+#### **--container**, **-c**
+
+By default `podman pod logs` retrives logs for all the containers available within the pod differentiate by field `container`. However there are use-cases where user would want to limit the log stream only to a particular container of a pod for such cases `-c` can be used like `podman pod logs -c ctrNameorID podname`.
```

Comment:
Why wouldn't I just use `podman logs CID?`, would `podman pod logs` give me different data?

Reply:
It majorly for  mainlining interface parity between `kubectl pod logs` and `podman pod logs` , since kubectl would not print logs if there are multiple containers in a pod it rather asks users to provide `container` using a `-c` flag.

Reply:
Ok